src/searchbar/SearchBar-default.js

```diff
@@ -4,162 +4,163 @@ import {
   ActivityIndicator,
   View,
   StyleSheet,
-  TextInput,
-  Platform,
-  Text as NativeText,
+  Dimensions,
+  Text,
 } from 'react-native';
 import colors from '../config/colors';
-import normalize from '../helpers/normalizeText';
+import renderIcon from '../helpers/renderIcon';
 import ViewPropTypes from '../config/ViewPropTypes';
-import getIconType from '../helpers/getIconType';
 
-class Search extends Component {
-  getRef = () => {
-    return this.input || this.refs[this.props.textInputRef];
+import Input from '../input/Input';
+
+const SCREEN_WIDTH = Dimensions.get('window').width;
+const defaultSearchIcon = {
```

Comment:
Same as above for defaultProps

Reply:
https://github.com/react-native-training/react-native-elements/pull/1135#discussion_r183971163